lib/pbench/server/indexing_tarballs.py

```diff
@@ -314,27 +317,28 @@ def process_tb(self, tarballs: List[TarballData]) -> int:
             idxctx.logger.info("Load templates {!r}", res)
             return res.value
 
-        idxctx.logger.debug("Preparing to index {:d} tar balls", len(tb_deque))
-
         with tempfile.TemporaryDirectory(
             prefix=f"{self.name}.", dir=idxctx.config.TMP
         ) as tmpdir:
-            idxctx.logger.debug("start processing list of tar balls")
+            usage = shutil.disk_usage(tmpdir)
+            idxctx.logger.info(
+                "start processing {:d} tar balls using TMP {} ({} free)",
```

Comment:
Are you sure you want this at `INFO` level?  (Are you even sure you want to commit this?)

Reply:
I put this in place to help diagnose the `ENOSPC` rather than simply hacking the live code in place. Yeah, since we solved that particular problem I could back this out; but while it does increase log noise it's the sort of clue that might prove useful again.